bonusy/datum/excs/prevod-casu.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Převod času
+demand: 1
+---
+
+V proměnné `apolloStart` máme uložený datum a čas startu Apolla 11. Vypiš datum na obrazovku ve formátu, na který jsou zvyklí Američané, tj. na první místo napiš měsíc, dále den a nakonec rok, jako oddělovače použij lomítka. Čas vypisovat nemusíš.
```

Comment:
Mohli bychom to rovnou sjednotit na snake case

```suggestion
V proměnné `apollo_start` máme uložený datum a čas startu Apolla 11. Vypiš datum na obrazovku ve formátu, na který jsou zvyklí Američané, tj. na první místo napiš měsíc, dále den a nakonec rok, jako oddělovače použij lomítka. Čas vypisovat nemusíš.
```

Reply:
Hromadný převod camel case -> snake case bychom mohli udělat pro všechno v separátním PR.

Reply:
Ok, tak já na to rovnou udělám issue: #15